charts/fluent-operator/Chart.yaml

```diff
@@ -23,10 +23,10 @@ maintainers:
     email: joshbaird@gmail.com
 dependencies:
   - name: fluent-bit-crds
-    repository: "file://charts/fluent-bit-crds"
-    version: 3.5.0
-    condition: fluentbit.crdsEnable
+    repository: "file://../charts/fluent-bit-crds" #  set to remote repository during release by CI
```

Comment:
The CI for releasing charts to fluent/helm-charts will change this `file://` reference to reference the actual repository.